miso-web/src/main/webapp/scripts/list_poolelement.js

```diff
@@ -71,104 +71,107 @@ ListTarget.poolelement = {
     return [];
   },
   createColumns: function(config, projectId) {
-    return [{
-      'sTitle': 'Dilution Name',
-      'mData': 'name',
-      'include': true,
-      'iSortPriority': 1
-    }, {
-      "sTitle": "Warnings",
-      "mData": null,
-      "mRender": WarningTarget.poolelement.tableWarnings(config.duplicateIndicesSequences, config.nearDuplicateIndicesSequences),
-      "include": true,
-      "iSortPriority": 0,
-      "bVisible": true,
-      "bSortable": false
-    }, {
-      'sTitle': 'Proportion',
-      'sType': 'numeric',
-      'mData': 'proportion',
-      'include': !config.add,
-      'iSortPriority': 0
-    }, ListUtils.idHyperlinkColumn("Library Name", "library", "library.id", function(dilution) {
-      return dilution.library.name;
-    }, 0, true, "noPrint"), ListUtils.labelHyperlinkColumn("Library Alias", "library", function(dilution) {
-      return dilution.library.id;
-    }, "library.alias", 0, true), ListUtils.idHyperlinkColumn("Sample Name", "sample", "library.parentSampleId", function(dilution) {
-      return "SAM" + dilution.library.parentSampleId;
-    }, 0, true, "noPrint"), ListUtils.labelHyperlinkColumn("Sample Alias", "sample", function(dilution) {
-      return dilution.library.parentSampleId;
-    }, "library.parentSampleAlias", 0, true, "noPrint"), {
-      'sTitle': 'Conc.',
-      'sType': 'numeric',
-      'mData': 'concentration',
-      'include': true,
-      'iSortPriority': 0,
-      'mRender': function(data, type, full) {
-        if (type === 'display' && !!data) {
-          var units = Constants.concentrationUnits.find(function(unit) {
-            return unit.name == full.concentrationUnits;
-          });
-          if (!!units) {
-            return data + ' ' + units.units;
+    return [
+        {
+          'sTitle': 'Dilution Name',
+          'mData': 'name',
+          'include': true,
+          'iSortPriority': 1
+        },
+        {
+          "sTitle": "Warnings",
+          "mData": null,
+          "mRender": Warning.tableWarningRenderer(WarningTarget.poolelement.makeTarget(config.duplicateIndicesSequences,
+              config.nearDuplicateIndicesSequences)),
```

Comment:
If the information about consent revoked is available here, could it be warned about as well?

Reply:
That's already done. The near/duplicate sequences are required in order to generate the WarningTarget because that information isn't available in the poolelement itself. Consent is available in the poolelement though, and the warnings are being displayed correctly as far as I can see?

Reply:
Oh, I see. And yes, looks like it's displaying correctly.